app/views/placements/wizards/add_user_wizard/_check_your_answers_step.html.erb

```diff
@@ -1,13 +1,19 @@
-<% caption = current_user.support_user? ? "#{t(".caption")} #{contextual_text}" : t(".caption") %>
-<%= content_for :page_title, sanitize(t(".page_title", caption:)) %>
+<%= content_for :page_title, sanitize(t(".page_title")) %>
 
 <%= form_for(check_your_answers_step, url: current_step_path, method: :put) do |f| %>
   <%= f.govuk_error_summary %>
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <span class="govuk-caption-l"><%= caption %></span>
-      <h1 class="govuk-heading-l"><%= t(".check_your_answers") %></h1>
+      <h1 class="govuk-heading-l"><%= t(".page_title") %></h1>
+
+      <% if @wizard.organisation.organisation_type == "school" %>
+        <p><%= t(".intro_text") %></p>
+      <% end %>
```

Comment:
Just a nitpick, but it might be nice to name this `school_intro_text` 🤔

```suggestion
      <% if @wizard.organisation.organisation_type == "school" %>
        <p><%= t(".school_intro_text") %></p>
      <% end %>
```